libvast/src/format/arrow.cpp

```diff
@@ -50,14 +49,16 @@ writer::~writer() {
 
 caf::error writer::write(const table_slice& slice) {
   if (out_ == nullptr)
-    return ec::filesystem_error;
+    return caf::make_error(ec::filesystem_error, "invalid arrow output stream");
   if (!layout(slice.layout()))
-    return ec::unspecified;
+    return caf::make_error(ec::logic_error, "failed to update layout");
```

Comment:
```suggestion
    return make_error(ec::logic_error, "failed to update layout");
```

Reply:
Not doing these. `vast::make_error` is just an alias for `caf::make_error`, and we use the latter more than the former in the code base.

Reply:
Just for the record:
```
$ git grep 'make_error(' | wc -l
343
$ git grep 'caf::make_error(' | wc -l
24
```

But it seems I have been outvoted on this issue.

Reply:
It's not as easy to grep for just error, but for `caf::error` the numbers were the other way around. ;)